decidim-bulletin_board-app/app/assets/javascripts/sandbox/vote.es6

```diff
@@ -9,6 +9,7 @@ $(async () => {
   const $castVote = $voter.find("button");
   const $vote = $voter.find("textarea");
   const $voterId = $voter.find("input");
+  const $doneMessage = $voter.find(".done-message");
 
   $vote.on("change", (event) => {
     $vote.css("border", "");
```

Comment:
as we now have a css file, maybe it would be better to use classes for these things

Reply:
> as we now have a css file, maybe it would be better to use classes for these things

I agree with your suggestion, but I prefer doing refactors in a separate test just to keep the PR for a single purpose. Actually, having a good e2e test suite will make those refactor a breeze 😄